db/data/20220707111908_backfill_degree_uuids.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class BackfillDegreeUuids < ActiveRecord::Migration[6.1]
+  def up
+    degrees = Degrees.all
+    degrees.each do |degree|
+      degree.update_columns(type_uuid: DfE::ReferenceData::Degrees::TYPES_INCLUDING_GENERICS.some({ name: degree.type }).first.id)
```

Comment:
I suggest invoking `update_columns` once for all columns and wrap the `ReferenceData` call in private methods.